src/rules/plugins/convert_sys_to_sys_exit.rs

```diff
@@ -0,0 +1,53 @@
+use rustpython_ast::{Expr, ExprKind};
+
+use crate::ast::types::{Binding, BindingKind, Range};
+use crate::autofix::Fix;
+use crate::check_ast::Checker;
+use crate::checks::{Check, CheckKind};
+
+fn has_builtin_exit_in_scope(checker: &Checker) -> bool {
+    matches!(
+        checker.current_scopes().find_map(|s| s.values.get("exit")),
+        Some(Binding {
+            kind: BindingKind::Builtin,
+            ..
+        })
+    )
+}
+
+fn get_exit_name(checker: &Checker) -> Option<String> {
+    checker.current_scopes().find_map(|s| {
+        s.values.values().find_map(|v| match &v.kind {
+            BindingKind::Importation(name, full_name, _) if full_name == &"sys".to_string() => {
+                Some(name.to_owned() + ".exit")
+            }
+            BindingKind::FromImportation(name, full_name, _)
+                if full_name == &"sys.exit".to_string() =>
+            {
+                Some(name.to_owned())
+            }
+            _ => None,
+        })
+    })
+}
```

Comment:
Probably a bit hard to get right since also needs to take aliases into account... but could be useful beyond this rule if robust.

Reply:
Do you mean alias as in `from a import b as alias` or `import a as alias`?
If yes, it already does.

Reply:
Yeah, that's what I meant! Oh, very nice.